testjs.js

```diff
@@ -0,0 +1,44 @@
+// 1. Programing Basics 
+
+const dishes = [
+    'Spaghetti',
+    'Lasagne',
+    'Beans on toast',
+    'Porridge',
+    'Tomato soup',
+    'Salmon'
+];
+
+for (let i = 0; i < dishes.length; i++) {
+    console.log(dishes[i]);
+}
+
+
```

Comment:
Good first part 👍🏽

The second part of the task is not done:
`2.Log out a randomly chosen dish (randomly generated index).`